qa/tasks/fwd_scrub.py

```diff
@@ -0,0 +1,209 @@
+"""
+Thrash mds by simulating failures
+"""
+import logging
+import contextlib
+import threading
+
+from gevent import sleep, killall, joinall, GreenletExit
+from gevent.greenlet import Greenlet
+from gevent.event import Event
+from teuthology import misc as teuthology
+from teuthology import contextutil
+
+from tasks import ceph_manager
+from tasks.cephfs.filesystem import MDSCluster, Filesystem
+from tasks.thrasher import Thrasher
+
+log = logging.getLogger(__name__)
+
+
+class MDSRankScrubber(Thrasher, Greenlet):
+    def __init__(self, fs, mds_rank, sleep_duration=30):
+        super(MDSRankScrubber, self).__init__()
+        self.logger = log.getChild('fs.[{f}]'.format(f=fs.name))
+        self.fs = fs
+        self.mds_rank = mds_rank
+        self.sleep_duration = sleep_duration
+
+    def _run(self):
+        try:
+            self.do_scrub()
+        except Exception as e:
+            self.set_thrasher_exception(e)
+            self.logger.exception("exception:")
+            # allow successful completion so gevent doesn't see an exception...
+
+    def _wait_until_scrub_complete(self, path="/", recursive=True):
+        tag = f"tag-{self.fs.name}-rank-{self.mds_rank}"
```

Comment:
I think a UUID might be the better approach here. We will probably want the scrubber to start scrubs at arbitrary subtrees at some point on the same MDS rank.

Reply:
since the default tag that gets assigned is indeed a UUID, I think we can use the same here without any issues